infra/gcp/terraform/modules/oci-proxy/variables.tf

```diff
@@ -23,11 +23,9 @@ variable "project_id" {
 variable "digest" {
   type = string
 }
-variable "cloud_run_config" {
-  type = map(object({
-    environment_variables = list(object({
-      value = string
-      name  = string
-    }))
-  }))
```

Comment:
I was hoping we could reuse this module to deploy the sandbox env. By removing this, testing specific configs in the sandbox env is not straightforward.

Reply:
It will be used to deploy the sandbox env. The sandbox env leads prod and should not skew on the cloud run config except explicit inputs. For example the verbosity is an explicit input to the cloud run config that varies between them.

You will be able to see the full diff by comparing the two root modules.

Reply:
https://github.com/BenTheElder/k8s.io/tree/module-staging / https://github.com/kubernetes/k8s.io/commit/a26bc4508c0995e0dc6eb8015487e6b266924752 cover staging

I split it out because this change is already large and that change is not quite complete (need to vet it carefully and write the tedious `moved` statements for everything).

But in particular staging vs prod:
https://github.com/kubernetes/k8s.io/blob/a26bc4508c0995e0dc6eb8015487e6b266924752/infra/gcp/terraform/k8s-infra-oci-proxy/main.tf

https://github.com/kubernetes/k8s.io/blob/a26bc4508c0995e0dc6eb8015487e6b266924752/infra/gcp/terraform/k8s-infra-oci-proxy-prod/main.tf